docs/config/README.md

```diff
@@ -3570,6 +3571,57 @@ The `pijul_channel` module shows the active channel of the repo in your current
 
+## Pixi
+
+The `pixi` module shows the installed [pixi](https://pixi.sh) version as well as the activated environment, if `$PIXI_ENVIRONMENT_NAME` is set.
+
+::: tip
+
+This does not suppress pixi's own prompt modifier, you may want to run `pixi config set change-ps1 false`.
+
+:::
+
+### Options
+
+| Option                     | Default                                                 | Description                                                                           |
+| -------------------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| `format`                   | `'via [$symbol($version )(\($environment\) )]($style)'` | The format for the module.                                                            |
+| `version_format`           | `'v${raw}'`                                             | The version format. Available vars are `raw`, `major`, `minor`, & `patch`.            |
+| `symbol`                   | `'📦 '`                                                 | The symbol used before the environment name.                                          |
+| `style`                    | `'yellow bold'                                          | The style for the module.                                                             |
+| `show_default_environment` | `true`                                                  | Whether to indicate that the `default` environment of your project is activated.      |
+| `pixi_binary`              | `['pixi']`                                              | Configures the python binaries that Starship should execute when getting the version. |
+| `detect_files`             | `['pixi.toml']`                                         | Which filenames should trigger this module                                            |
+| `disabled`                 | `false`                                                 | Disables the `conda` module.                                                          |
+
+### Variables
+
+| Variable    | Example   | Description                          |
+| ----------- | --------- | ------------------------------------ |
+| version     | `v0.33.0` | The version of `pixi`                |
+| environment | `py311`   | The current pixi environment         |
+| symbol      |           | Mirrors the value of option `symbol` |
+| style       |           | Mirrors the value of option `style`  |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[pixi]
+format = '[$symbol$environment](yellow) '
+
+[python]
+# customize python binary path for pixi
+python_binary = [
+  # this is the python from PATH if in a pixi shell
+  # (assuming you don't have python on your global PATH)
+  "python",
+  # fall back to pixi's python if it's available
+  ".pixi/envs/default/bin/python",
+]
```

Comment:
this is a bit hacky but works... not sure whether you want this workaround in your docs or not. i am also fine with removing it again